spec/services/foods_service_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe FoodsService do
+  it 'gets data for 10 foods based on keyword search' do
+    data = JSON.parse(File.read('spec/fixtures/cheese_data.json'), symbolize_names: true)
+    allow(FoodsService).to receive(:search_ingredient).and_return(data)
```

Comment:
When we are testing the service itself, we *don't* want to mock its return value, as that will make our test a bit useless (we're testing that the mock works rather than the actual logic in our service). So here, you'd want to use webmock or vcr because we *do* want to make sure that our service is receiving data from the external API, parsing the json, etc. Please reach out if you have any questions about this comment -- happy to chat.